### PR TITLE
Update indentation to match others

### PR DESCRIPTION
**Pull request by TPRobots**
Pull request opened by github-pullrequestcreator.

### CODE REVIEW

**Comment by b-dur** [comment]
Hi @martinbuberl. Could you CR this PR? I updated the indentation from 2 spaces to 4 spaces. Think that follows more the general code style in TP.

File: rules/eslint/style.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ module.exports = {
     // disallow if as the only statement in an else block
     'no-lonely-if': 1,
     // disallow mixed spaces and tabs for indentation
-    'no-mixed-spaces-and-tabs': [2, false],
+    'no-mixed-spaces-and-tabs': [4, false],
```
Comment (martinbuberl) [inline rules/eslint/style.js]: @b-dur We decided on 2 spaces for Node, which is the community/de-facto standard. That's why this is set to 2 here, as well as in several `.editorconfig`s. I'm aware of our 4 spaces rule for .NET projects.

**Comment by martinbuberl** [comment]
@b-dur Sorry for my delayed code review. As my comment above, we decided on 2 spaces for Node, which is the community/de-facto standard. That's why this is set to 2 here, as well as in several `.editorconfig`s. I'm aware of our 4 spaces rule for .NET projects.

**Comment by b-dur** [comment]
@martinbuberl where do I find this community/de-facto standard ?

**Comment by martinbuberl** [comment]
That's the IMHO community/de-facto standard: https://github.com/airbnb/javascript

**Pull request closed without merging**